git_trunk/git_trunk_commands.py

```diff
@@ -270,7 +270,7 @@ def handle_tracking_data(
                         args=(
                             trunk_tracking_data.remote,
                             trunk_tracking_data.head)
```

Comment:
If changing, then change this too:)
```suggestion
                        args=(
                            trunk_tracking_data.remote,
                            trunk_tracking_data.head
                        )
```